README.md

```diff
@@ -16,39 +46,125 @@ To start, you have to import libraries which will be used.
 ```javascript
 import initLogging, {
   createFirebaseLogger,
+  createCrashlyticsLogger,
   createSentryLogger,
+  setupReactotron,
   setupReactotronWithRedux,
   logEvent,
-} from 'react-native-logging';
+} from 'react-native-logging-tools';
 ```
 
-In this example, we will use reactotron with redux and firebase analytics.
+And the others libraries which can bu plugged
 ```javascript
 import Reactotron from 'reactotron-react-native';
-import {reactotronRedux} from 'reactotron-redux';
+import { reactotronRedux } from 'reactotron-redux';
 import analytics from '@react-native-firebase/analytics';
+import crashlytics from '@react-native-firebase/crashlytics';
 import * as Sentry from "@sentry/react-native";
+import AsyncStorage from '@react-native-community/async-storage';
 ```
 
-### How to use?
-Before any call to `react-native-logging`'s features, you have to initialize it.
+### How to use
+Before any call to `react-native-logging-tools`'s features, you have to initialize it.
+
+#### Initialization
+
+Librarie's initialization function take an object as parameter `initialization(init: IInit)`:
+init is an object which take three keys/values:
+- `config?: IConfig`: IConfig is an object which takes: 
```

Comment:
I would not put too much details in the usage section. I prefer keeping it as light as possible with a complete example. Then, I would create another "API documentation" section with a more detailed documentation of the initialization function and all the existing APIs